site/docs/v1/tech/admin-guide/troubleshooting.adoc

```diff
@@ -262,11 +262,19 @@ or
 org.elasticsearch.index.mapper.MapperParsingException: Could not dynamically add mapping for field [field.email]. Existing mapping for [data.field] must be of type object but found [text].
 ```
 
-This means that the type of a field in one of the data fields was changed. In this case, `data.field` was originally text, but the request causing this error message was trying to put an object into that attribute.
+This means that the type of a field in one of the data fields was changed.
+In this case, `data.field` was originally text, but the request causing this error message was trying to put an object into that attribute.
+This can interfere with user imports, searches and more.
 
-Elasticsearch will dynamically build a schema based upon custom data, so it is critical that the schema not be modified.
+FusionAuth uses Elasticsearch to dynamically build the schema, so it is critical that the schema not be modified manually.
```

Comment:
FusionAuth doesn't do anything here.. it is all Elasticsearch. We don't tell it to do this, just want to make that clear. This sounds like the way we chose to do it this way, which maybe you can argue is true, but it seems like it is more accurate to say this is what Elasticsearch does.

Reply:
I'll see if I can reword it. I wanted to convey that we rebuild the index, including the schema, every time we re-index. I believe that is what you said yesterday, and that's what the code says. That means that any changes someone makes are transient.

Maybe

> The schema is reset every time you issue a [re-index command](https://fusionauth.io/docs/v1/tech/apis/system#rebuild-the-elasticsearch-index) so manual changes will not be persisted.